slides/05-feature-engineering.qmd

```diff
@@ -488,7 +488,7 @@ What happens for a 20% threshold?
 #| echo: false
 
 glm_val_pred %>% 
-  roc_curve(truth = on_goal, estimate = .pred_yes) %>% 
```

Comment:
Maybe these have been fixed already. I don't see how they passed in previous sessions.